.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
This tells [`nvm`](https://github.com/creationix/nvm#nvmrc) to use the latest LTS release, currently 10.13.0 – I've also updated the package.json to reflect this as the node version.